flash/vision/style_transfer/data.py

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
I need an image dataset preferably unsupervised here. The original model was trained on the COCO 2014 train split.

Reply:
@pmeier consider to look at https://github.com/PyTorchLightning/lightning-flash/pull/256 since the api for loading data will change quite a bit